win32ss/gdi/ntgdi/freetype.c

```diff
@@ -163,6 +163,42 @@ static const CHARSETINFO FontTci[MAXTCIINDEX] =
 /* list head */
 static RTL_STATIC_LIST_HEAD(FontSubstListHead);
 
+PRFONT
+NTAPI
+RFONT_AllocRFONT(void)
+{
+    PRFONT prfnt;
+
+    /* Allocate the RFONT structure */
+    prfnt = ExAllocatePoolWithTag(PagedPool, sizeof(RFONT), GDITAG_TEXT);
```

Comment:
What about using "sizeof(*prfnt)" instead to not hardcode the variable type? (same comment for the RtlZeroMemory call below).

Reply:
If the variable type changes it wouldn't be clled prfnt as well.

Reply:
...but you can easily refactor it then.